pom.xml

```diff
@@ -5,7 +5,7 @@
     <modelVersion>4.0.0</modelVersion>
 
     <groupId>com.github.sisyphsu</groupId>
-    <artifactId>dateparser</artifactId>
+    <artifactId>dateparser-xyzt-ai</artifactId>
```

Comment:
Ow, this was a mistake.
This was done for our local fork, and should not have been included in this PR. Will revert this for the PR.